components/navigation/learningItems.tsx

```diff
@@ -0,0 +1,19 @@
+import IconGradCap from '../icons/GradCap';
+import IconGuide from '../icons/Guide';
+import IconMigration from '../icons/Migration';
+import IconPaper from '../icons/Paper';
+import IconPlant from '../icons/Plant';
+import IconRocket from '../icons/Rocket';
+import IconUsers from '../icons/Users';
+
+const learningItems = [
```

Comment:
define types for the element inside the array, in this component file. Applicable for all files.